udi-core/src/main/java/no/nav/registre/core/database/model/Person.java

```diff
@@ -5,8 +5,11 @@
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
+import lombok.ToString;
 import no.nav.registre.core.database.model.opphold.OppholdStatus;
 import no.udi.mt_1067_nav_data.v1.JaNeiUavklart;
+import org.hibernate.annotations.Fetch;
```

Comment:
Skal ikke være nødvendig å bruke hibernate sine annotasjoner her

Reply:
Viste seg at om man ikke gjør det slik stenger Hibernate sin connection ned før transaksjonen har blitt fullført, så jeg trenger å beholde denne.

Reply:
Legg kanskje til en kommentar i koden for hvorfor det er brukt spesifikke imports her. Kan nok være fort gjort for neste person å fjerne disse

Reply:
Lurt, lagt til forklarende kommentar nå